stores/directory/local.py

```diff
@@ -1,8 +1,11 @@
 from __future__ import annotations
```

Comment:
annotations. and the. abc imports shouldn't be in here, think they've crept in elsewhere.